vhdbuilder/release-notes/AKSUbuntu/gen2/1804gpucontainerd/latest.txt

```diff
@@ -854,7 +867,7 @@ Disk usage:
 Filesystem      Size  Used Avail Use% Mounted on
 udev             55G     0   55G   0% /dev
 tmpfs            11G  704K   11G   1% /run
-/dev/sda1        29G   26G  3.3G  89% /
+/dev/sda1        29G   27G  2.6G  92% /
```

Comment:
space filling up again, might need to address this for next week

Reply:
I saw that. it's because we are getting more and more Kubernetes versions,  (new cube-proxy, new kubelet / Kubectl,e tc)